test_suite/conftest.py

```diff
@@ -14,7 +14,7 @@ def check_markers(host, request):
     skip_message = "This test doesn't apply to {distro_version}"
 
     def parse_distro_version(distro_version):
-        return float(re.findall(r"\d+(\.\d+)?", distro_version)[0])
+        return float((re.search(r"\d+(\.\d+)?", distro_version)).group(0))
```

Comment:
Could this bring issues when there's actually only major version provided?
![image](https://user-images.githubusercontent.com/100693208/217543932-dbde6e9b-277f-45b9-abfe-afb8ecdbd2df.png)
E.g. for fedora, which does not include minor version.

Reply:
This won't raise issues:
If we want to exclude rhel or fedora35 distro for example you don't need to use the operators < >, so this function won't be call.

Reply:
You're right! Thanks for the great explanation. In that case all good from my side about this PR :+1: